src/server/lib/middleware/requestState.ts

```diff
@@ -41,6 +46,11 @@ const getRequestState = (req: RequestWithTypedQuery): RequestState => {
   const browser = Bowser.getParser(req.header('user-agent') || 'unknown');
 
   return {
+    rateLimitData: {
+      email: formEmail || encryptedStateEmail,
+      ip: req.ip,
+      accessToken: req.cookies.SC_GU_U,
+    },
```

Comment:
Wondering if this needs to be set on the request, seems to duplicate info that is already available on the request object, and I don't think it's used except for in the Redis middleware?

Reply:
yes, I think you're right - it should be localised to the Redis middleware as it's not used anywhere else!